core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3558,6 +3559,28 @@ private void setAttributeFromEntry(SetAttributeEntry entry)
     return mBlockMaster.getWorkerInfoList();
   }
 
+  @Override
+  public Map<String, Long> getMetrics() {
+    MetricRegistry mr = MetricsSystem.METRIC_REGISTRY;
+    Map<String, Long> metricsMap = new HashMap<>();
+
+    for (Map.Entry<String, Counter> entry : mr.getCounters().entrySet()) {
```

Comment:
so will this only collect the metrics registered in default fs master?

Reply:
Not really, also includes metrics like ufsSessionCount. I changed to meta_master.thrift.